#include <compass/MsgCompassAngle.h>
#include <compass/MsgCompassAcceleration.h>
#include <compass/MsgCompassAngleRate.h>
#include <navig/MsgEstimatedPosition.h>
#include <dvl/MsgDistance.h>
#include <dvl/MsgVelocity.h>
#include <dvl/MsgHeight.h>
#include <gps/MsgGpsCoordinate.h>
#include <gps/MsgGpsSatellites.h>
#include <gps/MsgGpsUtc.h>
#include <supervisor/MsgDepth.h>